### ISSUE TITLE
"LuaSkin: Unable to fetch element attribute NSAccessibilityFrontmostAttribute for: ..."

### ISSUE THREAD

**Issue by roeybiran** [Hammerspoon/hammerspoon#2573]
Hello,
Ever since upgrading to 0.9.80 I've been getting these weird message in the console. The are of all the same form:
`2020-10-23 10:50:18: 10:50:18 ERROR:   LuaSkin: Unable to fetch element attribute NSAccessibilityFrontmostAttribute for: X`
Where `X` is the name of a given app. These messages appear constantly. At first I thought it's coming from my `hs.application.watcher` or `hs.window.filter` instances when they try to query a non-native app, but sadly I'm getting this for nearly every app on my Mac, including native ones!

Thanks, and I simply ❤️ Hammerspoon

**Comment by asmagill** [comment]
The latest version is actually 0.9.81 (https://github.com/Hammerspoon/hammerspoon/releases) and this logging message is suppressed in that version.

**Comment by roeybiran** [comment]
Thank you so much! I wonder why I didn't get the auto-update prompt.
